### PR TITLE
Rename test file to make it discoverable by pytest

### PR DESCRIPTION
**Pull request by ukleinek**
This way the test program is picked up by calling pytest without any
parameters.

The motivation for this is that discoverable tests are run automatically
by the helper scripts during Debian package generation.

### CODE REVIEW

**Review by open-dynaMIX — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution @ukleinek :tada: 

Could you please rebase this PR onto current `main`?

File: tox.ini [hunk 1]
```diff
@@ -13,7 +13,7 @@ envlist = py{27,36,37,38,39,310}, pypy3, flake8, black
 deps=
     pytest
     pytest-cov
-commands=pytest -r a -vv tests.py anonip.py
+commands=pytest -r a -vv test_module.py anonip.py
```
[the remark this answers is not in the document]
Comment (ukleinek) [reply tox.ini:16]: Might be, the changes to `tox.ini` where mechanical, didn't test anything and I'm a fan of minimal changes. So following the rename there seemed a``` good idea to me.

[the remark this answers is not in the document]
Comment (ukleinek) [reply tox.ini:16]: For sure dropping `anonip.py` there would need some research on my side to convince me that's ok.

**Comment by ukleinek** [comment]
> Could you please rebase this PR onto current `main`?

Looking at the history I wonder about this request. I assume you would merge using `--no-ff` anyhow, so this cannot be about a linear history which some project favour. My PR merges just fine into current `main`, so I don't understand the problem that solves. I only see some downsides. (Additional work, throwing away the current state being (somewhat) tested, ...)

File: tox.ini (hunk 1 above)
[the remark this answers is not in the document]
Comment (open-dynaMIX) [reply tox.ini:16]: I'm fine with keeping your change.

**Comment by open-dynaMIX** [comment]
The reason for me asking you to rebase was because of the merge conflict due to changes to `tests.py` earlier today.

**Comment by open-dynaMIX** [comment]
Closed in favor of #71

**Comment by ukleinek** [comment]
> The reason for me asking you to rebase was because of the merge conflict due to changes to `tests.py` earlier today.

git handles the merge conflict just fine for me:

```
uwe@taurus:~/debpkg/anonip/Anonip$ git checkout 5f9578496260
HEAD is now at 5f9578496260 Merge pull request #70 from open-dynaMIX/fix_mixed_case_ipv6

uwe@taurus:~/debpkg/anonip/Anonip$ git pull --no-edit https://github.com/ukleinek/Anonip discoverable-test
From https://github.com/ukleinek/Anonip
 * branch                      discoverable-test -> FETCH_HEAD
Merge made by the 'ort' strategy.
 tests.py => test_module.py | 0
 tox.ini                    | 2 +-
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename tests.py => test_module.py (100%)

uwe@taurus:~/debpkg/anonip/Anonip$ git grep C0A8 test_module.py
test_module.py:        ("[2001:db8:1::ab9:C0A8:102]:8080", 12, 84, "[2001:db8::]:8080"),
```

So maybe you just experienced github being less potential than the original git?! *shrug*

**Pull request closed without merging**